src/model/snake.py

```diff
@@ -0,0 +1,61 @@
+from typing import Tuple, List
+from enum import Enum
+
+
+class Direction(Enum):
+    """Enum class representing directions."""
+
+    RIGHT = 0
+    LEFT = 1
+    UP = 2
+    DOWN = 3
+
+
+class Snake:
+    """Representation of the Snake entity."""
+
+    def __init__(self, position: Tuple[int, int], direction: Direction):
+        self.position = position
+        self.direction = direction
+        self.body = self._create_body(position)
+
+    def _create_body(self, position: Tuple[int, int]) -> List[Tuple[int, int]]:
+        body = [position]
+        body.extend((body[i - 1][0] - 10, position[1]) for i in range(1, 4))
```

Comment:
I just took from the GeeksForGeeks example. In my mind, it should be `1` instead of `10`. Maybe they use `10` in that tutorial for some specific reason due to coupling. I will investigate.